plugins/workspace-search/src/workspace_search.ts

```diff
@@ -0,0 +1,581 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Class responsible for workspace search.
```

Comment:
You can remove this part (fileoverview is optional, we removed authors from most places already)

Reply:
Done